From 224862d4788aadda4775fc3991d836d7233e9528 Mon Sep 17 00:00:00 2001
From: Dheeraj CVR <cvr.dheeraj@gmail.com>
Date: Mon, 23 Jun 2014 22:00:07 +0530
Subject: [PATCH 102/211] Camera2: Prevent propogating CancelAutoFocus during
 Video Recording

When AutoFocus is started during Video Recording, CancelAutoFocus
is called after KEY_FOCUS_TIME seconds. CancelAutoFocus has the
side effect of resetting the sensor AF position and will result in
instant loss of focus which is not a desired event when AF is
manually triggered in case of Video Recording.

Change-Id: Icee7f6dfd4ad083c9b383ee2a4df6a34e4668de6
---
 src/com/android/camera/VideoModule.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/com/android/camera/VideoModule.java b/src/com/android/camera/VideoModule.java
index 6fcbf1181..180e3ad03 100644
--- a/src/com/android/camera/VideoModule.java
+++ b/src/com/android/camera/VideoModule.java
@@ -600,7 +600,6 @@ public class VideoModule implements CameraModule,
     @Override
     public void cancelAutoFocus() {
         if (null != mCameraDevice) {
-            mCameraDevice.cancelAutoFocus();
             setFocusParameters();
         }
     }
-- 
2.17.1

